lib/health-checker-middleware.js

```diff
@@ -1,9 +1,31 @@
-module.exports = function ({ uaString = '' }) {
-  return function healthChecker({ headers }, res, next) {
-    let { 'user-agent':ua = '' } = headers;
-    if (ua.match(uaString)) {
-      return res.sendStatus(200);
+const STRATEGIES = {
+  UA_SNIFF(uaString) {
+    return function({ headers }, res, next) {
+      let { 'user-agent':ua = '' } = headers;
+      if (ua.match(uaString)) {
+        return res.sendStatus(200);
+      }
+      return next();
     }
-    return next();
+  },
+  PATH_HANDLER: (_req, res) => res.sendStatus(200),
+}
+
+function findStrategy(options = {}) {
+  if (options.uaString || options.strategy === 'ua-sniff') {
+    return STRATEGIES.UA_SNIFF(options.uaString || options.string);
+  }
+
+  if (options.path || options.strategy === 'path') {
```

Comment:
if `options.path` is set to the `path` argument passed to `app.use()` on index.js L#41, then this PR LGTM. If not, I'm not sure why this OR statement is needed as opposed to just looking for the `strategy`

Reply:
yeah, you're right. I left it in from a previous iteration. will remove.